examples/tokens/ERC1155ownable.vy

```diff
@@ -345,6 +352,7 @@ def setURI(uri: String[MAX_URI_LENGTH]):
     self._uri = uri
     log URI(uri, 0)
 
+@view
 @external
 def uri(id: uint256) -> String[MAX_URI_LENGTH]:
```

Comment:
Not really sure why there's a `contractURI` when this function exists

Reply:
The contract uri points to a JSON metadata file describing the NFT collection. 
the 'regular URI' points to individual NFT metadata files.
Without the contractURI there is no collection description and image visible on NFT marketplaces.

Reply:
Hope this explains things a bit 😊

Reply:
This isn't a part of the standard, is it?

Reply:
Pure erc1155. no.
Name and Symbol are also not part of the pure standard.

But: 
With name, symbol and the contract metadata URI this example is opensea.io compatible.
I think this may help Vyper adoption.
(Also working on a brownie mix that includes this contract, and setting up to do a comparison between this contract and a solidity 1155 to see what the gas usage difference is.)

Reply:
The latter. This is an opensea "thing".

So for name, symbol, contract URI explicitly add to the documentation these are opensea.io additions?

Reply:
Documented the opensea specific functions